### PR TITLE
Run scroll command via Angular afterRender hook

### LINKED ISSUE #1121: Chat component's auto-scroll not working when used with a web component
**Describe the bug**
The auto-scroll from the chat component is not working when a web component is added after the chat component.

**To Reproduce**

1. Clone the repo from https://github.com/TheFirstMe/mesop-fastapi-chat-bug/
2. Start the application Go to http://127.0.0.1:8000/
3. Enter any message in the chat
4. See error in console

**Expected behavior**
After user enters the message, the auto-scroll behaviour should take place.

**Screenshots**
Image

**Desktop System Info**

- OS: Ubuntu
- Browser Microsoft Edge
- Version 22.04

**Additional context**
- The Mesop app is mounted to a FastAPI application.
- The Mesop app code used can be found here: https://github.com/TheFirstMe/mesop-fastapi-chat-bug/blob/main/app/ui/main.py
    ```python
    @me.page(
        security_policy=me.SecurityPolicy(
            allowed_script_srcs=["https://cdn.jsdelivr.net"],
            allowed_iframe_parents=["https://google.github.io", "https://huggingface.co"],
        ),
        path="/",
        title="Mesop Demo Chat",
        on_load=on_load,
    )
    def page():
        mel.chat(transform, title="Mesop Demo Chat", bot_user="Mesop Bot")
        test_component() # Removing this line fixes the issue
    ```
- This issue does not happen when I remove the web component from the page.

### PR DESCRIPTION
**Pull request by richard-to**
Finally got a chance to test out the afterRender suggestion. It seems to work.

Closes #1121

### CODE REVIEW

File: mesop/web/src/shell/shell.ts [hunk 1]
```diff
@@ -257,6 +241,34 @@ export class Shell {
...
+  maybeExecuteScrollCommand() {
+    if (this.commandScrollKey) {
+      const targetElements = document.querySelectorAll(
+        `[data-key="${this.commandScrollKey}"]`,
+      );
+      if (!targetElements.length) {
+        console.error(
+          `Could not scroll to component with key ${this.commandScrollKey} because no component found`,
+        );
+        return;
```
Comment (wwwillchen) [inline mesop/web/src/shell/shell.ts:255]: probably would be good to do this before the return:
`this.commandScrollKey = '';` 

this branch is already an edge case, but having the command scroll key somehow run in a future render loop when the the target element exists, seems like an even more bizzare edge case.

**Review by wwwillchen — APPROVED** [review]
Just checking, were you able to run a playwright test N times, just to see whether the flakiness issue with scrolling is solved?

File: mesop/web/src/shell/shell.ts (hunk 1 above)
Comment (richard-to) [reply mesop/web/src/shell/shell.ts:255]: Ah yeah good call.

**Comment by richard-to** [comment]
> Just checking, were you able to run a playwright test N times, just to see whether the flakiness issue with scrolling is solved?
> 
> Also, neat that this worked! Thanks.

Good idea. I ran it for 10 times just now and I didn't run into any flakiness with the scrolling.
